test/pytest/test_code.py

```diff
@@ -15,26 +15,22 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import os
-import pytest
-import shutil
 import subprocess
 
-
-ROOT_DIR = os.path.realpath(f'{__file__}/../../..')
+import pytest
 
 
-@pytest.mark.skipif(not shutil.which('mypy'), reason='mypy is not installed')
-def test_bridge_mypy():
+@pytest.mark.parametrize('command', [
     # only test src/cockpit; src/systemd_ctypes does not have type annotations yet
     # disable caching, as it otherwise often crashes with "Cannot find component 'inotify' for 'systemd_ctypes...."
-    subprocess.check_call(['mypy', '--no-incremental', 'src/cockpit'], cwd=ROOT_DIR)
-
+    ['mypy', '--no-incremental', 'src/cockpit'],
 
```

Comment:
Unnecessary newline

Reply:
I did this because otherwise it was difficult to understand that comment above applied only to the one line.